addons/test_mail/static/tests/activity_tests.js

```diff
@@ -415,13 +415,15 @@ QUnit.module("test_mail", {}, function () {
             "No template should be available"
         );
 
-        await click(document.querySelector(".o-activity-list-popover-add-activity-button"));
+        await click($(".o-activity-list-popover button:contains(Schedule an activity)")[0]);
```

Comment:
Isn't it the `click` that accepts a jquery selector?

Reply:
This is the `click` of web, which I think is not jquery behind the hood compared to mail `click`